index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <h4>Hello world we are developers</h4>
+<h1 id="title">Survey form </h1>
+  <p id="description"> Lorem ipsum dolor sit amet consectetur
+  </p>
+<form id="survey-form">
+  <label for="name" id="name-label">Name</label>
+    <input id="name" placeholder="name" required>
+    <label for="email" id="email-label">Email</label>
+    <input type="email" id="email" placeholder="email" required>
+    <label for="number" id="number-label">Number</label>
+    <input type="number" id="number" min="7" max="15" required placeholder="number">
+    <select id="dropdown" name="dropdown">
+      <option value="">Number</option>
+      <option> Number</option>
```

Comment:
Kindly remove the empty spaces.

Reply:
- [ ] Kindly always remember to mention the reason why the reviewee had to do these changes so please add the reason.
for this example you can say:
`Please remove the white spaces to make your code clean and more readable`

Reply:
- [ ] Kindly implemented the changes asked by the previous reviewer. Make sure to replace your comment `Kindly remove the empty spaces.` with the suggested one `Please remove the white spaces to make your code clean and more readable` .